charts/primary-site/templates/deployments/inbox-listener.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: inbox-listener
+  labels:
+    app: foxglove-inbox-listener
+spec:
+  selector:
+    matchLabels:
+      app: foxglove-inbox-listener
+  strategy:
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: foxglove-inbox-listener
+    spec:
+      volumes:
+        - name: cloud-credentials
+          secret:
+            secretName: gcp-cloud-credential
+            optional: true
+      containers:
+        - name: inbox-listener
+          image: us-central1-docker.pkg.dev/foxglove-images/images/inbox-listener:dcb5b4812e4d0a6054638ea43712f2e7ee5050ab
+          resources:
+            requests:
+              cpu: 1000m
+              memory: 1Gi
+            limits:
+              cpu: 1000m
+              memory: 1Gi
```

Comment:
I think we should have requests/limits parameterized in values files, rather than set here

Reply:
Agreed - we can do that in follow-up improvements to the charts